docs/design/software-requirements-specification.adoc

```diff
@@ -224,22 +224,25 @@ a| The task shall build a Python application.
 [cols="1,3"]
 |===
 | SRS-TASK-BUILD-NPM-1
-a| The task shall run test, creating code coverage and xUnit reports.
+| The task shall run the NPM lint script.
```

Comment:
Minor thing:
It was written like this before in the docs as I see but when you look at [npm.js](https://www.npmjs.com/) it should be `npm` instead of `NPM`.

Reply:
Thanks, fixed.